infrastructure/interface/launch/machine_robb.launch

```diff
@@ -0,0 +1,80 @@
+  <arg name="marker_length" default="0.164"/>
+  <arg name="markers_topic_name" default="/$(arg curr_namespace)/markers"/>
+  <arg name="rotation_offset" default="[0.0,0.0,0.0]"/>
+  <arg name="front_corner" default="0"/>
+
+  <arg name="marker_world_translation" default="0 0 0"/>
+  <!-- <arg name="marker_world_quat" default="0 0 0.7071068 0.7071068"/> -->
+  <!-- <arg name="marker_world_quat" default="0 0 0 1"/> -->
+  <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="tf_world"  -->
+    <!-- args="$(arg marker_world_translation) $(arg marker_world_quat) marker_$(arg main_marker) $(arg world_frame)" /> -->
+    
+  <group ns="$(arg curr_namespace)">
+
+    <!-- Manager nodelet -->
+    <node pkg="nodelet" type="nodelet" name="standalone_nodelet"  args="manager" output="screen"/>
+
+    <!-- Camera reader -->
+    <node pkg="nodelet" type="nodelet" name="physical_camera_nodelet_t" output="screen"
+      args="load perception/physical_camera_nodelet_t standalone_nodelet" >
+      <rosparam command="load" file="$(arg camera_config)" />
+    </node>      
+
+    <!-- Aruco detection -->
+    <node pkg="nodelet" type="nodelet" name="aruco_detection_nodelet_t" output="screen"
+      args="load perception/aruco_detection_nodelet_t standalone_nodelet" >
+      <rosparam command="load" file="$(find perception)/config/aruco_detection.yaml" />
+    </node>
+
+    <!-- camera To world -->
+    <node pkg="nodelet" type="nodelet" name="aruco_cTw_nodelet_t" output="screen"
+      args="load estimation/aruco_cTw_nodelet_t standalone_nodelet">
+      <rosparam command="load" file="$(arg camera_config)" />
+      <param name="front_corner"        value="$(arg front_corner)" />
+      <param name="main_marker"         value="$(arg main_marker)" />
+      <param name="camera_frame"         value="$(arg camera_frame)" />
+      <param name="marker_length"       value="$(arg marker_length)" />
+      <param name="camera_id"       value="1" />
+      <param name="markers_topic_name"  value="$(arg markers_topic_name)" />
+      <param name="robot_id"            value="$(arg robot_id)" />
+      <rosparam param="rotation_offset" subst_value="True">$(arg rotation_offset)</rosparam>
+    </node>  
+  
+    <!-- world To camera -->
+    <node pkg="nodelet" type="nodelet" name="aruco_wTc_nodelet_t" output="screen"
+      args="load estimation/aruco_wTc_nodelet_t standalone_nodelet" >
+      <rosparam command="load" file="$(arg camera_config)" />
+      <param name="image_topic" value="/$(arg curr_namespace)/camera/rgb"/>
+      <param name="goal_pose_topic" value="/mushr/feedback_goal_pos"/>
+      <param name="goal_rad_topic" value="/mushr/feedback_goal_rad"/>
+      <param name="trajectory_topic" value="$(arg trajectory_topic)"/>
+      <param name="camera_frame" value="C1"/>
+      <param name="world_frame" value="$(arg world_frame)"/>
+      <param name="robot_frame" value="$(arg robot_frame)"/>
+
+    </node>   
+
+    <!-- Plant estimator -->
```

Comment:
Nit: commented out logic that we probably don't need since we are modifying the plant estimator to just get the pose from the markers for now.